packages/angular/projects/clr-angular/src/data/datagrid/datagrid-column-toggle.ts

```diff
@@ -38,14 +38,12 @@ import { ClrDatagridColumnToggleButton } from './datagrid-column-toggle-button';
     <div
       class="column-switch"
       role="dialog"
+      [attr.aria-label]="commonStrings.keys.showColumnsMenuDescription"
       [id]="popoverId"
       clrFocusTrap
       *clrPopoverContent="openState; at: smartPosition; outsideClickToClose: true; scrollToClose: true"
     >
       <div class="switch-header">
-        <div class="clr-sr-only" tabindex="-1" #menuDescription>
-          {{ commonStrings.keys.showColumnsMenuDescription }}
-        </div>
```

Comment:
Where does this and related removals come from? I can't find them as part of the linked issue.

Reply:
Previously, the description for the column toggle panel was being done with this invisible div that was programatically focused when the column chooser was opened.  The problem with this is that while it's read when the panel is opened, when the user exits the panel the screen reader will tell them that they have exited a dialog but not what dialog it was that they exited.  This change moves the description to the dialog itself so that it's read both when the user enters it and leaves it.  

There's still another one of these invisible divs in the dialog after this change, but a better way of handling that would probably to use the Angular CDK's aria-live service rather than trying to get the same effect by shifting focus to an invisible div.

Reply:
Sounds good. Btw, we're staying away from making our own aria-live implementations or services, because we don't want to impose any such implementations to the user. So, your solution is more in sync with our current strategy. I like it when code gets removed.